setup.py

```diff
@@ -66,6 +66,7 @@
     "invenio-accounts>=1.3.0",
     "invenio-files-rest>=1.2.0",
     "invenio-records-permissions>=0.8.0",
+    "xmltodict~=0.12.0",
```

Comment:
We can "pilot" using tilda `~` requirements, and review on a per-package basis what's the best practice (e.g. in this case we would trust `xmltodict 0.12.*`)

Reply:
:+1: 

To be clear (and I ranted about this before :sweat_smile:), I am in favour of putting lids on dependency versions now. Otherwise we have to constantly react to upstreams breaking.